jenkins-job-generator/definitions/linux-package-jobs.yml

```diff
@@ -22,22 +26,6 @@
                 label 'agent-ubuntu-local'
             }
             stages {
-                stage("Checkout Infrastructure Repository") {
```

Comment:
Here the Infrastructure Repo checkout is still needed.

Reply:
Really? Here in linux-package-jobs.yml we don't build anymore. We build directly in build-jobs.yml line (line 537) . The only steps which follow this line are copying deb files and recreate_deb_repository.py

Reply:
makes sense ...